daemon/graphdriver/overlay2/overlay.go

```diff
@@ -191,11 +192,11 @@ func Init(home string, options []string, uidMaps, gidMaps []idtools.IDMap) (grap
 		// Try to enable project quota support over xfs.
 		if d.quotaCtl, err = quota.NewControl(home); err == nil {
 			projectQuotaSupported = true
+		} else if opts.quota.Size > 0 {
+			return nil, fmt.Errorf("Project quota is not supported: %v", err)
```

Comment:
Made the change as suggested. The messages are little different but mean the same.